<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cellular Automata</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StepButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="StepSaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save step?</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="XCoordinateText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="XCoordinateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X coordinate</string>
    </property>
   </widget>
   <widget class="QLabel" name="YCoordinateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y coordinate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="YCoordinateText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="NewButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadScriptButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Script</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ScriptText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>881</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ScriptLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>981</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Script Status: Not loaded</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>1011</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveScriptButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
